psi-sample/enclave/build.gradle

```
@@ -0,0 +1,42 @@
+plugins {
+    id 'com.r3.conclave.enclave'
+}
+
+dependencies {
+    implementation "com.r3.conclave:conclave-enclave"
+    testImplementation "com.r3.conclave:conclave-testing"
+    testImplementation "org.junit.jupiter:junit-jupiter:5.6.0"
+    implementation project(":common");
```

for consistency with the client module I think it would be better to put the dependency on `common` first. I also think it's nice to separate the test dependencies
```suggestion
    implementation project(":common");

    implementation "com.r3.conclave:conclave-enclave"
    
    testImplementation "com.r3.conclave:conclave-testing"
    testImplementation "org.junit.jupiter:junit-jupiter:5.6.0"
```

---

done